### PR TITLE
Add more debug msg when helm upgrade

### PR DESCRIPTION
**Pull request by yxxhero**
**What this PR does / why we need it**:
add more debug msg when upgrade. https://github.com/helm/helm/issues/10183
**Special notes for your reviewer**:
add more debug msg when upgrade. when helm upgrade with --debug, there is  no message about `patch`.
**If applicable**:
- [ ] this PR contains documentation
- [ ] this PR contains unit tests
- [ ] this PR has been tested for backwards compatibility

### CODE REVIEW

**Comment by yxxhero** [comment]
@hickeyma what do you think of? This PR will print more meesage to help debug upgrade issue.

File: pkg/kube/client.go [hunk 1]
```diff
@@ -264,7 +264,7 @@ func (c *Client) Update(original, target ResourceList, force bool) (*Result, err
 	}
 
 	for _, info := range original.Difference(target) {
-		c.Log("Deleting %q in %s...", info.Name, info.Namespace)
+		c.Log("Deleting %s called %q in Namespace %s...", info.Mapping.GroupVersionKind.Kind, info.Name, info.Namespace)
```
Comment (hickeyma) [inline pkg/kube/client.go:267]: I think it would read better as: `Deleting %s %q in namespace %s...`

File: pkg/kube/client.go [hunk 2]
```diff
@@ -507,6 +507,7 @@ func updateResource(c *Client, target *resource.Info, currentObj runtime.Object,
 			return nil
 		}
 		// send patch to server
+		c.Log("Patch %q with kind %s in Namespace %s", target.Name, kind, target.Namespace)
```
Comment (hickeyma) [inline pkg/kube/client.go:510]: I think it would read better as: `Patch %s %q in namespace %s`

**Review by hickeyma — CHANGES_REQUESTED** [review]
Thanks for the PR @yxxhero. Some small nits suggested.

**Comment by yxxhero** [comment]
@hickeyma done. Thanks very much.

File: pkg/kube/client.go [hunk 3]
```diff
@@ -507,6 +507,7 @@ func updateResource(c *Client, target *resource.Info, currentObj runtime.Object,
 			return nil
 		}
 		// send patch to server
+		c.Log("Patch %s %q in namespace %s", target.Name, kind, target.Namespace)
```
Comment (hickeyma) [inline pkg/kube/client.go:510]: c.Log("Patch %s %q in namespace %s", target.Name, kind, target.Namespace) --> c.Log("Patch %s %q in namespace %s", kind, target.Name, target.Namespace)

**Review by hickeyma — CHANGES_REQUESTED** [review]
Nearly there. .. just order mixed up for one msg

File: pkg/kube/client.go (hunk 3 above)
Comment (yxxhero) [reply pkg/kube/client.go:510]: oh, sorry. I will change it.

**Review by hickeyma — COMMENTED** [review]
Ok, looks good @yxxhero. Do you mind squashing the commits first before I merge?

**Comment by yxxhero** [comment]
@hickeyma done. thanks very much.

**Review by hickeyma — APPROVED** [review]
LGTM, thanks @yxxhero

**Pull request merged**